packages/confirm-dialog/src/vaadin-confirm-dialog.js

```diff
@@ -75,13 +75,18 @@ class ConfirmDialog extends ConfirmDialogMixin(ElementMixin(ThemePropertyMixin(P
       :host([opened]),
       :host([opening]),
       :host([closing]) {
-        display: contents !important;
+        display: block !important;
+        position: absolute;
       }
 
       :host,
       :host([hidden]) {
         display: none !important;
       }
+
+      :host(:focus) ::part(overlay) {
+        outline: var(--vaadin-focus-ring-width) solid var(--vaadin-focus-ring-color);
```

Comment:
By default this only applies to base styles, as Lumo resets both of these properties:

https://github.com/vaadin/web-components/blob/8b4ca330ada97a7d20f43f2818cebe8359a3d58d/packages/vaadin-lumo-styles/src/props/reset.css#L8-L10

IMO it's ok as in V24, there is no default focus indicator on the overlay part in Lumo.